core/aop-runtime/test/fixtures/modules/hello_cross_cut/HelloCrossCut.ts

```
@@ -0,0 +1,69 @@
+  cut: Math.random().toString(),
+};
+
+@Crosscut({
+  type: PointcutType.CLASS,
+  clazz: Hello,
+  methodName: 'hello',
+}, { adviceParams: crosscutAdviceParams })
+@Advice({
+  accessLevel: AccessLevel.PUBLIC,
+})
+export class CrosscutAdvice implements IAdvice<Hello, String> {
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)